packages/consoledot-api/src/transformers/consumerTransformer.ts

```diff
@@ -0,0 +1,25 @@
+import { ConsumerGroupState } from '@rhoas/kafka-instance-sdk';
+import type { ConsumerGroup as ConsumerGroupApi } from "@rhoas/kafka-instance-sdk/dist/generated/model/consumer-group";
+import type { ConsumerGroup} from "ui-models/src/models/consumer-group";
+
+export function consumerTransformer(t: ConsumerGroupApi): ConsumerGroup {
+      const state = stateMapping[t.state as ConsumerGroupState]
+      return {
+        groupId: t.groupId || "",
```

Comment:
Don't default to empty string. Throw an error if the data we require to be there isn't there.